extensions/barrage/src/test/java/io/deephaven/extensions/barrage/chunk/BarrageColumnRoundTripTest.java

```diff
@@ -570,14 +571,15 @@ private static <T> void testRoundTripSerialization(
                 final ArrayList<ChunkInputStreamGenerator.FieldNodeInfo> fieldNodes = new ArrayList<>();
                 column.visitFieldNodes((numElements, nullCount) -> fieldNodes
                         .add(new ChunkInputStreamGenerator.FieldNodeInfo(numElements, nullCount)));
-                final TLongArrayList bufferNodes = new TLongArrayList();
+                final LongStream.Builder bufferNodes = LongStream.builder();
```

Comment:
This might be a good use case for a RowSetBuilderSequential or RowSetBuilderRandom.

Reply:
They are definitely ordered and sorted. These are offsets into the payload of where to find each buffer.

Reply:
Agreed. It's a test anyway, not worth the effort.